src/Datadog.Trace.Tools.Runner/Program.cs

```diff
@@ -0,0 +1,141 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading;
+using CommandLine;
+using CommandLine.Text;
+
+namespace Datadog.Trace.Tools.Runner
+{
+    internal class Program
+    {
+        private static CancellationTokenSource _tokenSource = new CancellationTokenSource();
+
+        private static string RunnerFolder { get; set; }
+
+        private static Platform Platform { get; set; }
+
+        private static void Main(string[] args)
+        {
+            // Initializing
+            string executablePath = (Assembly.GetEntryAssembly() ?? Assembly.GetExecutingAssembly()).Location;
+            string location = executablePath;
+            if (string.IsNullOrEmpty(location))
+            {
+                location = Environment.GetCommandLineArgs().FirstOrDefault();
+            }
+
+            string executableName = Path.GetFileNameWithoutExtension(location).Replace(" ", string.Empty);
```

Comment:
Is this variable used for anything?

Reply:
No, removed in https://github.com/DataDog/dd-trace-dotnet/pull/962/commits/63917cc110edc33e13efdfc3076cb207fe537d54